pontoon/base/templates/project_selector.html

```diff
@@ -26,7 +26,7 @@
         {% endif %}
           <span class="name"
             data-slug="{{ p.slug }}"
-            data-details="{{ p.details }}">{{ p.name }}</span>
+            {% if p.details %}data-details="{{ p.details }}"{% endif %}>{{ p.name }}</span>
```

Comment:
From what I can tell from the JS if no project details are found, `Pontoon.getProjectDetails` will return null and code in `translate_old.js` will fail as it's not set up to handle null being returned.

Why is this being removed, and how does the JS not fail when it is?